gr-digital/grc/variable_header_format_default.block.yml

```diff
@@ -20,25 +20,25 @@ value: ${ digital.header_format_default(access_code, threshold, bps) }
 templates:
     imports: from gnuradio import digital
     var_make: |-
-        % if int(eval(access_code))==0:
-        self.${id} = ${id} = digital.header_format_default(digital.packet_utils.default_access_code,\
+        % if access_code.isdigit():
+        self.${id} = ${id} = digital.header_format_default(${access_code},\
         ${threshold}, ${bps})
         % else:
-        self.${id} = ${id} = digital.header_format_default(${access_code},\
+        self.${id} = ${id} = digital.header_format_default(digital.packet_utils.default_access_code,\
         ${threshold}, ${bps})
         % endif
 
 cpp_templates:
     includes: ['#include <gnuradio/digital/header_format_default.h>']
     declarations: 'digital::header_format_default::sptr ${id};'
     var_make: |-
-        % if int(eval(access_code))==0:
-        this->${id} = ${id} = digital::header_format_default(digital.packet_utils.default_access_code,\
+        % if access_code.isdigit():
+        this->${id} = ${id} = digital::header_format_default(${access_code},\
         ${threshold}, ${bps});
         % else:
-        this->${id} = ${id} = digital::header_format_default(${access_code},\
+        this->${id} = ${id} = digital::header_format_default(digital.packet_utils.default_access_code,\
```

Comment:
This doesn't look like it'd produce valid C++. We'd have to `${..}` the python.

Reply:
@marcusmueller You are right, this does not produce valid c++ code, though this is the same code as before just in an other order.
At this level you can't evaluate digital.packet_utils.default_access_code as the digital package is not known here.

But I found out that there are some more issues to be solved in grc.
See #2688 
There happens a KeyError for variable blocks without a value param when trying to build the cpp code. This is similar to #2662 for python code.
I try to find a fix.